pages/pages.json

```diff
@@ -0,0 +1,120 @@
+{
+  "routes": {
+    "store": {
```

Comment:
This route is already defined in `vtex.store`

Reply:
It causes bugs if it's not declared here. Before I didn't have that, and it caused the home page to render blank, because the `__children__` prop for the LayoutContainer was undefined. This is the only solution that solved the problem.